drivers/middleware/stdlib/middleware.go

```diff
@@ -33,9 +33,32 @@ func NewMiddleware(limiter *limiter.Limiter, options ...Option) *Middleware {
 	return middleware
 }
 
+// NewJWTMiddleware return a new instance of a JWT token middleware.
+func NewJWTMiddleware(limiter *limiter.Limiter, options ...Option) *Middleware {
```

Comment:
Please remove this function because it doesn't provide any values.

You could define KeyGetter with your JWTKeyGetter by declaring an option.
You can find inspiration on how to do this there: 
https://github.com/ulule/limiter/blob/c3db9d65aeed3f147a7b45e0ba412cf958e11dc8/drivers/middleware/stdlib/options.go#L53-L58

Reply:
Then, you could declare your middleware with something like this: 

```go
stdlib.NewMiddleware(limiter, stdlib.WithJWTKeyGetter(JWTKeyGetter(limiter)))
```

Or, simply do a function that does something like this if you want syntactic sugar for this.